steps/go-template/go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/awrenn/relay-twilio-transformer
```

Comment:
I'd probably go ahead and update this as well (not that it really makes much difference).

Reply:
:+1: Thanks, missed that one!